src/PlaywrightSharp/Browser.cs

```diff
@@ -8,6 +8,10 @@
 
 namespace PlaywrightSharp
 {
+    public partial interface IBrowser : IAsyncDisposable
+    {
+    }
```

Comment:
We can add this to the `IBrowser` interface you created in contracts.

Reply:
Nice catch, completely forgot I had that - I added it *before* I realized I needed an overload :-).